libs/utils/decorate.js

```diff
@@ -1,6 +1,3 @@
-import { createTag } from './utils.js';
```

Comment:
createTag is used in decorate,js file. I believe we shouldn't be removing it's import as this will cause error. It is also causing failure of unit tests
<img width="880" alt="Screen Shot 2023-09-13 at 10 40 25 PM" src="https://github.com/adobecom/milo/assets/69535463/1aa87d3b-7446-4bff-a363-56b03e9ff72a">

Reply:
yeah, it seems odd to me that it's importing from decorate, but safer to restore it